### PR TITLE
Add download button

### LINKED ISSUE #6351: Allow admin users to download workspaces
We once had a "Download workspace content" button on the admin dashboard which was quite handy when trying to analyze abuse. Would be nice to have it again!

missing-button

- [x] Add download button to admin dashboard
- [ ] Adjust the [workspace download route config](https://github.com/gitpod-io/gitpod/issues/6351#issuecomment-982373095)

### PR DESCRIPTION
**Pull request by laushinka**
## Description
- Adds download button to the admin dashboard
- Removes the reload button

Screenshot 2021-11-29 at 13 26 43

## Related Issue(s)

Relates #6351

## How to test
When I test on staging, it gives me an error. Is there a way I can test this?

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **laushinka** after the PR has been reviewed.

Associated issue: *#6351*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by gtsiolis — APPROVED** [review]
UX works like a charm, @laushinka! 🔮 

> @laushinka: When I test on staging, it gives me an error. Is there a way I can test this?

FYI, The error occurs when you try to download a running workspace but this is out of the scope of these changes, see https://github.com/gitpod-io/gitpod/issues/4977.

---

Approving to unblock merging but holding in case @geropl or @jankeromnes want to take a closer look at the code changes. 🏀 

/hold

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 5afa9b0174aeb8d32c6f55c4a888c9af24aa93f0</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6962#pullrequestreview-818267448" title="Approved">gtsiolis</a>*

Associated issue: *#6351*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [gtsiolis]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by geropl — APPROVED** [review]
The code LGTM as well.

But note that this alone won't fix #6351: The route `/workspace-download/get/<wsId>` does not work on environments that don't use minio. Will elaborate in https://github.com/gitpod-io/gitpod/issues/6351.

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6962#pullrequestreview-818660469" title="Approved">geropl</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/6962#pullrequestreview-818267448" title="Approved">gtsiolis</a>*

Associated issue: *#6351*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [geropl,gtsiolis]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
